migrations/2024-07-27-212955_add_token_locker_id_in_payment_method/down.sql

```diff
@@ -0,0 +1,3 @@
+-- This file should undo anything in `up.sql`
+
+ALTER TABLE payment_methods DROP COLUMN IF EXISTS network_network_token_locker_id;
```

Comment:
Is this supposed to be `network_token_locker_id`?

```suggestion
ALTER TABLE payment_methods DROP COLUMN IF EXISTS network_token_locker_id;
```